build.sh

```diff
@@ -143,11 +149,12 @@ pixman_dependencies=""
 cairo_dependencies="zlib png pixman"
 xml_dependencies="zlib iconv"
 sqlite_dependencies=""
-openslide_dependencies="png jpeg tiff openjpeg glib gdkpixbuf cairo xml sqlite"
+openslide_dependencies="libzip png jpeg tiff openjpeg glib gdkpixbuf cairo xml sqlite"
 openslidejava_dependencies="openslide"
 
 # Build artifacts
 zlib_artifacts="zlib1.dll"
+libzip_artifacts="libzip.dll"
```

Comment:
Changed name to "libzip-4.dll", made test runs on XP 32 and Vista 32.
Perhaps the code at _openslide_zip_open_archive( ) needs reinvestigation, because it's now possible to create a zip_source from a FILE* handle under Windows.